encrypt_util.go

```diff
@@ -83,7 +83,6 @@ func encryptStream(
 		}
 		mode.CryptBlocks(cipherText, chunk)
 		out.Write(cipherText[:len(chunk)])
```

Comment:
Just noticed: this should probably be `out.Write(cipherText[:n])` to more proactively prevent writing buffered data from previous loop cycles.